### PR TITLE
Items index

### PR DESCRIPTION
**Pull request by sciencefixion**
@jKwibe @RyN21  This is user story 17, finally completed! I'm not sure I needed what I added to the application css, but it took me a while to figure out how to target the image links in my spec. I welcome any feedback and thank you for your seemingly infinite patience!

### CODE REVIEW

File: spec/features/items/index_spec.rb [hunk 1]
```diff
@@ -57,5 +58,31 @@
         expect(page).to have_css("img[src*='#{@dog_bone.image}']")
       end
     end
+
```
Comment (RyN21) [inline spec/features/items/index_spec.rb:61]: Nice test!

File: app/views/items/index.html.erb [hunk 2]
```diff
@@ -6,20 +6,24 @@
...
-        <p>Active</p>
-      <% else %>
-        <p>Inactive</p>
-      <% end %>
-    </section>
+    <% if item.enabled? %>
+      <section class = "grid-item" id= 'item-<%=item.id%>'>
+        <h2> <%=link_to item.name, "/items/#{item.id}" %> </h2>
+        <p>Sold by: <%=link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
+        <div class="image-link" id= 'link-<%= item.id %>'>
```
Comment (RyN21) [inline app/views/items/index.html.erb:13]: Nice work with working with the images!

File: app/views/items/index.html.erb [hunk 3]
```diff
@@ -10,8 +10,9 @@
       <section class = "grid-item" id= 'item-<%=item.id%>'>
         <h2> <%=link_to item.name, "/items/#{item.id}" %> </h2>
         <p>Sold by: <%=link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
-        
-        <img src= <%= item.image %>>
+        <div class="image-link" id= 'link-<%= item.id %>'>
+          <%= link_to(image_tag("#{item.image}", :alt => "#{item.name}", :title => "#{item.name}"), "/items/#{item.id}") %>
```
Comment (jKwibe) [inline app/views/items/index.html.erb:14]: I like this a lot.

File: db/schema.rb [hunk 4]
```diff
@@ -36,6 +36,7 @@
     t.bigint "merchant_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.boolean "enabled?", default: true
```
Comment (jKwibe) [inline db/schema.rb:39]: Nice boolean.

File: app/assets/stylesheets/application.css [hunk 5]
```diff
@@ -14,6 +14,10 @@
  *= require_self
  */
 
+ .image-link {
+
```
Comment (jKwibe) [inline app/assets/stylesheets/application.css:18]: You did not need this if you are not styling.

**Pull request merged**